src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -41,6 +41,7 @@ const DiningBalance = () => {
   } else if (typeof diningInfo !== 'object') {
     //Set color to use when displaying balance based on how low it is...
     const diningBalance = parseInt(diningInfo);
+    let balanceColor = colors.secondary.green;
```

Comment:
This line probably shouldn't have been added, colors.secondary.green doesn't exist.